docs/protocol/economic-security/eigenlayer-avs.mdx

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
noting the title here is "Hyperlane AVS" but the filename and what it shows up as in the contents on the left is "EigenLayer AVS". Not sure if intentional but flagging in case - im happy as-is fwiw

Reply:
I didn't rename the file and left as is, I think it would make sense to make the naming consistent. It is called EigenLayer AVS is the sidebar. What do we think, Hyperlane AVS?